packages/react/src/breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { extractSprinkles } from "../sprinkles";
+import * as styles from "./Breadcrumb.css";
+import { useBreadcrumbItems } from "./useBreadcrumbItems";
+
+export type BreadcrumbProps = BoxProps<
+  "nav",
+  {
+    children: React.ReactNode;
```

Comment:
```suggestion
    children: ReactNode;
```
We can remove the redundant `React` and import `ReactNode` from named export.